src/components/SearchHeader.js

```diff
@@ -27,7 +29,8 @@ export default class SearchHeader extends React.PureComponent {
   }
 
   search = () => {
-    console.log('search called');
+    const {query, searchBy} = this.state;
```

Comment:
I would use destructuring props and state by default :)

Reply:
I am not sure I understand what you mean here. You mean in the rest of the files I should use destructuring more? or in this case in particular there is something wrong with the destructuring?